autogen/main/README.md

```diff
@@ -261,6 +261,9 @@ The node_pools variable takes the following parameters:
 | queued_provisioning | Makes nodes obtainable through the ProvisioningRequest API exclusively. | | Optional |
 | gpu_sharing_strategy | The type of GPU sharing strategy to enable on the GPU node. Accepted values are: "TIME_SHARING" and "MPS". | | Optional |
 | max_shared_clients_per_gpu | The maximum number of containers that can share a GPU. | | Optional |
+| consume_reservation_type | The type of reservation consumption. Accepted values are: "UNSPECIFIED": Default value. This should not be used, "NO_RESERVATION": Do not consume from any reserved capacity, "ANY_RESERVATION": Consume any reservation available, "SPECIFIC_RESERVATION": Must consume from a specific reservation. Must specify key value fields for specifying the reservations. | | Optional |
+| reservation_affinity_key | The label key of a reservation resource. To target a SPECIFIC_RESERVATION by name, specify "compute.googleapis.com/reservation-name" as the key and specify the name of your reservation as its value. | | Optional |
+| reservation_affinity_values | The list of label values of reservation resources. For example: the name of the specific reservation when using a key of "compute.googleapis.com/reservation-name". This should be passed as comma separated string. | | Optional |
```

Comment:
Would it be possible to use a list of strings? `["value1", "value"2"]` otherwise might want to later `trimspace()` to address if they are comma+space separated.

Reply:
cant, but `trimspace` makes sense

Reply:
Sure - I think a `trimspace` after the `split("," ...)` will guard against "value1, value2" accidentally ending up as ["value1", " value2"]